src/VirtualClient/VirtualClient.Core/UnixDiskManager.cs

```diff
@@ -169,6 +169,45 @@ public override Task<IEnumerable<Disk>> GetDisksAsync(CancellationToken cancella
             });
         }
 
+        /// <summary>
+        /// Grabs the available data directory on the system.
+        /// </summary>
+        /// <param name="cancellationToken"></param>
+        /// <param name="diskFilter">Disk filter to </param>
+        /// <returns></returns>
+        /// <exception cref="WorkloadException"></exception>
+        public override Task<IEnumerable<string>> GetDiskPathsAsync(string diskFilter, CancellationToken cancellationToken)
+        {
+            List<string> diskPaths = new List<string>();
+
+            return this.Logger.LogMessageAsync($"{nameof(UnixDiskManager)}.GetDiskPaths", EventContext.Persisted(), async () =>
```

Comment:
You only need this wrap if you care about start and ends, do you need that information?